build-gadget.sh

```diff
@@ -2,11 +2,14 @@
 
 # Remove the pc-gadget directory if it already exists
 rm -rf pc-gadget
-git clone https://github.com/snapcore/pc-gadget.git --branch=22 --depth 1 
 
-# Build gadget snap
+git clone https://github.com/snapcore/pc-gadget.git
```

Comment:
Why is the branch removed? It is still useful to make it clear that this is based on the core22 gadget:
```suggestion
git clone https://github.com/snapcore/pc-gadget.git --branch=22
```

Reply:
Because they explicitly check out the same commit, the end result would be the same. 

But I understand your point. I will apply your suggestion, thanks.